docs/_reporting/hubs/README.md

```diff
@@ -38,10 +38,23 @@ FinOps hubs are a reliable, trustworthy platform for cost analytics, insights, a
+
+    Scalability: The ADX integration is designed to handle larger data volumes, scaling up capacity with adjustable VM sizes. The default setup can manage up to $10M/month, making it ideal for larger organizations.
+
+    Built-In Queries: ADX offers pre-configured queries that simplify cost exploration, letting you dive into granular data views without extensive setup.
+
+For a more detailed view and featre comparison between the different Hubs versions, refer to the [help me choose](../power-bi/README.md)page.
+
 <blockquote class="highlight-green-title" markdown="1">
-  💵 Estimated cost: $25/mo per $1M in cost being monitored
+  💵 Estimated cost: 
   
-  _Estimated cost includes $5 for Azure storage and data processing plus up to $20 per user for [Power BI licenses](https://www.microsoft.com/power-platform/products/power-bi/pricing). Exact cost will vary based on discounts, data size per $1M (~20GB of data), and Power BI license requirements. Pipelines will run once a day per export, plus one additional monthly run per export. Pipeline run time depends on data size. For details, refer to the [FinOps hub cost estimate](https://azure.com/e/c3d98263bec048b6af52acb180c42b7e) in the Azure Pricing Calculator or monitor hub cost using the [Data ingestion report](../power-bi/data-ingestion.md)._
+  Hubs with Azure Data Factory:  **$25/mo per $1M in cost being monitored**
+    _Estimated cost includes $5 for Azure storage and data processing plus up to $20 per user for [Power BI licenses](https://www.microsoft.com/power-platform/products/power-bi/pricing). Exact cost will vary based on discounts, data size per $1M (~20GB of data), and Power BI license requirements. Pipelines will run once a day per export, plus one additional monthly run per export. Pipeline run time depends on data size. For details, refer to the [FinOps hub cost estimate](https://aka.ms/finops/hubs/calculator) in the Azure Pricing Calculator or monitor hub cost using the [Data ingestion report](../power-bi/data-ingestion.md)._
+
+  Hubs with Azure Data Explorer:  ** ~$220 per month**
+    _Estimated cost includes The estimated cost of Hubs on ADX assumes instance is running 24x7. Costs can be reduced by enabling auto-shutdown outside of business hours or applying commitment discounts. Prices are estiamted in UK South as of October 2024. If you are using Power BI dashboards, you need to include up to $20 per user for [Power BI licenses](https://www.microsoft.com/power-platform/products/power-bi/pricing). Exact cost will vary based on discounts, data size per $1M (~20GB of data), and Power BI license requirements. Pipelines will run once a day per export, plus one additional monthly run per export. Pipeline run time depends on data size. For details, refer to the [FinOps hub cost estimate](https://aka.ms/finops/hubs/calculator/adx) in the Azure Pricing Calculator or monitor hub cost using the [Data ingestion report](../power-bi/data-ingestion.md)._
```

Comment:
Fwiw, the previous block of text was already too big. This is making it even bigger. Should we move this to a section further down? It's not enough for a separate doc.